### PR TITLE
Release Candidate 3.8.0

### PR DESCRIPTION
**Pull request by Buwujiu**
- Feature to allow song_analysis index request fetch analysis payload from kafka message instead of song.
- Jenkins file update.

### CODE REVIEW

File: maestro-domain/pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>maestro</artifactId>
         <groupId>bio.overture</groupId>
-        <version>${revision}${sha1}${changelist}</version>
+        <version>3.8.0</version>
```
Comment (blabadi) [inline maestro-domain/pom.xml:26]: You don't need to do this in Maestro, you change the .mvn/conf file for the version

File: pom.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
     <groupId>bio.overture</groupId>
     <artifactId>maestro</artifactId>
     <!-- from .mvn/maven.config -->
-    <version>${revision}${sha1}${changelist}</version>
```
Comment (blabadi) [inline pom.xml:32]: same as above the version is in a config file

File: maestro-domain/pom.xml (hunk 1 above)
Comment (Buwujiu) [reply maestro-domain/pom.xml]: reverted

File: pom.xml (hunk 2 above)
Comment (Buwujiu) [reply pom.xml]: reverted

**Review by blabadi — APPROVED** [review]

**Pull request merged**